charts/business-central/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   # image.repository -- Repository of the image to use. You can use a generic image and set `image.artifactUrl` or use a pre-built image and set `image.artifactUrl` to `""`. See [here](#using-artifacts-and-pre-built-images) for more information. around the two options.
-  repository: mcr.microsoft.com/dynamicsnav
+  repository: mcr.microsoft.com/businesscentral
   # image.tag -- Tag of the image to use
   tag: 10.0.17763.1397-generic
```

Comment:
I think we need to remove `-generic` here as well according to the new naming

Reply:
Yeah you were right. I removed the -generic tag

Reply:
Hm, I think I fixed all the linting errors but I don't know how to get passed Validating Maintainer 404 error.

Btw I realized that the Generic Image is to old for the win nodes OS... I added accept outdated but is there anyway we could use the Get-BestGenericImageName or something to dynamically update the generic image version to the current winver on the azure vms?

Reply:
just digged into the linting. I think the  maintainer name in the `chart.yaml` needs to be a valid GitHub name. Could you change that to `lippertmarkus` to see if that works? Strange behaviour as it did work before with my real name..

According to the generic image version: Automatic determination is not very trivial. You could just use `kubectl get node -o wide` to get the kernel version of your Windows workers and then add `--set image.tag "10.0.17763.1757"` with your kernel version.